### DISCUSSION TITLE
Whether files can be backed up by date ?

### DISCUSSION THREAD

**Discussion by ts-liushiyuan** [megahertz/electron-log#288] [Q&A]
Whether files can be backed up by date ?

**Comment by megahertz** [comment]
It's possible to set a custom handler for `log.transports.file.archiveLog` or [`log.transports.file.resolvePath`](https://github.com/megahertz/electron-log/blob/master/docs/file.md#resolvepath-variables-pathvariables-message-logmessage--string) depending on your requirements.
